src/Uno.UI/UI/Xaml/Controls/Primitives/RepeatButton/RepeatButton.mux.cs

```diff
@@ -0,0 +1,377 @@
+		{
+			int newDelay = (int)pNewDelay;
+			if (newDelay < 0)
+			{
+				throw new ArgumentOutOfRangeException(nameof(Delay), "Delay cannot be less than 0.");
+			}
+		}
+
+		private void OnIntervalPropertyChanged(object pNewInterval)
+		{
+			int newInterval = (int)pNewInterval;
+			if (newInterval <= 0)
+			{
+				throw new ArgumentOutOfRangeException(nameof(Interval), "Interval cannot be less than 0.");
+			}
+		}
+
+		/// <summary>
+		/// Called when the IsEnabled property changes.
+		/// </summary>
+		/// <param name="e">Event args.</param>
+		private protected override void OnIsEnabledChanged(IsEnabledChangedEventArgs e)
+		{
+			base.OnIsEnabledChanged(e);
+
+			_keyboardCausingRepeat = false;
+			_pointerCausingRepeat = false;
+
+			UpdateRepeatState();
+		}
+
+		/// <summary>
+		/// KeyDown event handler.
+		/// </summary>
+		/// <param name="args">Event args.</param>
+		protected override void OnKeyDown(KeyRoutedEventArgs args)
+		{
+			if (args.Key == VirtualKey.Space &&
+				ClickMode != ClickMode.Hover)
+			{
+				_keyboardCausingRepeat = true;
+				UpdateRepeatState();
+			}
+
+			base.OnKeyDown(args);
+		}
+
+		/// <summary>
+		/// KeyUp event handler.
+		/// </summary>
+		/// <param name="args">Event args.</param>
+		protected override void OnKeyUp(KeyRoutedEventArgs args)
+		{
+			base.OnKeyUp(args);
+
+			if (args.Key == VirtualKey.Space &&
+				ClickMode != ClickMode.Hover)
+			{
+				_keyboardCausingRepeat = false;
+				UpdateRepeatState();
+			}
+		}
+
+		/// <summary>
+		/// LostFocus event handler.
+		/// </summary>
+		/// <param name="e">Event args.</param>
+		protected override void OnLostFocus(RoutedEventArgs e)
+		{
+			base.OnLostFocus(e);
+
+			if (ClickMode != ClickMode.Hover)
+			{
+				_keyboardCausingRepeat = false;
+				_pointerCausingRepeat = false;
+				UpdateRepeatState();
+			}
+		}
+
+		private bool ShouldIgnoreInput(PointerRoutedEventArgs args)
+		{
+			bool ignoreInput = false;
+
+			if (IgnoreTouchInput)
+			{
+				var pointerPoint = args.GetCurrentPoint(null);
+				var pointerDeviceType = pointerPoint.PointerDevice?.PointerDeviceType ?? PointerDeviceType.Touch;
+				if (pointerDeviceType == PointerDeviceType.Touch)
+				{
+					ignoreInput = true;
+				}
+			}
+
+			return ignoreInput;
+		}
+
+		/// <summary>
+		/// PointerEnter event handler.
+		/// </summary>
+		/// <param name="args">Event args.</param>
+		protected override void OnPointerEntered(PointerRoutedEventArgs args)
+		{
+			if (ShouldIgnoreInput(args))
+			{
+				return;
+			}
+
+			base.OnPointerEntered(args);
+
+			if (ClickMode == ClickMode.Hover)
+			{
+				_pointerCausingRepeat = true;
+			}
+
+			UpdateRepeatState();
+			UpdateVisualState();
+		}
+
+		/// <summary>
+		/// PointerMoved event handler.
+		/// </summary>
+		/// <param name="args">Event args.</param>
+		protected override void OnPointerMoved(PointerRoutedEventArgs args)
+		{
+			// The reason this function does not call the base 
```

Comment:
This tends to confirm that the `_pointerPosition` field in `ButtonBase` is pretty useless as we don't even let the base class update it.

Reply:
Making the field `private` only, but will still keep the code which works with it, as it is ported. Maybe this will be adjusted later in WinUI as well